libraries/Microsoft.Bot.Connector/ConnectorClient.cs

```diff
@@ -325,7 +325,7 @@ private void Initialize()
             BaseUri = new System.Uri("https://api.botframework.com");
             SerializationSettings = new JsonSerializerSettings
             {
-                Formatting = Newtonsoft.Json.Formatting.Indented,
+                Formatting = Newtonsoft.Json.Formatting.None,
```

Comment:
is there any tests can help verify this settings? or you can paste your manual test result?

Reply:
Update the description with locale test